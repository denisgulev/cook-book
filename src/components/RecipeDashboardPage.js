import React, { Fragment } from "react";
import RecipeList from "./RecipeList";
import RecipeListFilters from "./RecipeListFilters";
import RecipesSummary from "./RecipesSummary";

const RecipeDashboardPage = () => (
  <Fragment>
    <RecipesSummary />
    <RecipeList />
  </Fragment>
);

export default RecipeDashboardPage;
