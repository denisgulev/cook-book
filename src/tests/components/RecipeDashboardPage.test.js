import React from "react";
import { shallow } from "enzyme";
import RecipeDashboardPage from "../../components/RecipeDashboardPage";

test("should render RecipeDashboardPage correctly", () => {
  const wrapper = shallow(<RecipeDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
