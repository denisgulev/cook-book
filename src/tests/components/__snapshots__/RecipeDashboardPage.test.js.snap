// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render RecipeDashboardPage correctly 1`] = `
<div>
  <Connect(RecipesSummary) />
  <Connect(RecipeListFilters) />
  <Connect(RecipeList) />
</div>
`;
